libbpf-cargo/src/btf/btf.rs

```diff
@@ -206,6 +210,18 @@ impl<'a> Btf<'a> {
                     btf::BtfIntEncoding::Char | btf::BtfIntEncoding::None => format!("u{}", width),
                 }
             }
+            BtfType::Float(t) => {
+                let width = match t.size {
+                    2 => bail!("Unsupported float width"),
+                    4 => "32",
+                    8 => "64",
+                    12 => bail!("Unsupported float width"),
+                    16 => bail!("Unsupported float width"),
+                    _ => bail!("Invalid float width"),
+                };
```

Comment:
question: why isn't 2 => "16" and 16 => "128"?

Reply:
Rust itself don't have `f16` and `f128` :)